layouts/partials/openapi/render-object-table.html

```diff
@@ -97,6 +98,26 @@
 
     {{ end }}
 
+ {{/* Additional properties should either be an OpenAPI schema object, or a
+       boolean. If it's the former, look for a `description` and render it as a note.
+       If it's `true`, do nothing. If it's `false`, raise a build-time error:
+       "Everything is extensible in Matrix unless otherwise noted". */}}
+   {{ if (ne $additionalProperties nil )}}
+  <tr class="row-additional-properties">
+   <td colspan="3">
```

Comment:
it seems odd that we include the table row and empty cell if `additionalProperties` is true?

Reply:
Yes. A previous version of this PR used to have the cell say "May have additional properties" in that situation. Looks like I didn't clean up the HTML for when I removed that case.